pages/linux/mdbook.md

```diff
@@ -0,0 +1,20 @@
+# mdbook
+
+> Create online books by writing makrdown files.
+> More information: <https://rust-lang.github.io/mdBook/>.
+
+- Init mdbook files in  dir, default current dir:
```

Comment:
Do this command create a `mdbook` project? Maybe this can say:
```suggestion
- Create a mdbook project in the current directory:
```
and you can split the `mdbook init` and `mdbook init {{path/to/directory}}` in 2 examples

Reply:
Hey @hellojukay, this wasn't changed. Do you need more help to apply this?